maya.py

```diff
@@ -53,29 +40,45 @@ def __format__(self, *args, **kwargs):
         """Return's the datetime's format"""
         return format(self.datetime(), *args, **kwargs)
 
-    @validate_type_mayadt
+    def _validate_type_mayadt(operator):
```

Comment:
There is no reason to move this decorator into the `MayaDT` class. I'd suggest to leave it outside the class.